packages/navie/src/agents/gatherer.ts

```diff
@@ -126,10 +150,19 @@ To gather the information, respond with the following commands:
 !!find <path> [-depth <depth>]
 !!cat <path>[:<start-line>[-<end-line>]]
 
-If you don't need any further information, respond with !!finish on a single line.
-Remember, it's not your job to address the question, just to gather the information. Be thorough.`;
+You can also do a full-text search using
+!!search <search terms>
```

Comment:
I guess this will also trigger a rebuild of the index? If we are seeing Navie perform multiple context searches per question, it would start to make sense to have a minimal amount of index caching (at the least, cache per user message).